managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/OpFindNewest.java

```diff
@@ -15,13 +15,14 @@
  */
 package org.apache.bookkeeper.mledger.impl;
 
-import org.apache.bookkeeper.mledger.AsyncCallbacks.ReadEntryCallback;
 import org.apache.bookkeeper.mledger.AsyncCallbacks.FindEntryCallback;
+import org.apache.bookkeeper.mledger.AsyncCallbacks.ReadEntryCallback;
 import org.apache.bookkeeper.mledger.Entry;
 import org.apache.bookkeeper.mledger.ManagedLedgerException;
 import org.apache.bookkeeper.mledger.Position;
 import org.apache.bookkeeper.mledger.impl.ManagedLedgerImpl.PositionBound;
-import com.google.common.base.Predicate;
+
+import java.util.function.Predicate;
```

Comment:
in general we should switch to JDK's own things when available, though in this case this is unrelated so I'll take it out